app/templates/rules.html

```diff
@@ -84,7 +84,12 @@ <h3 class="panel-title">
     <div class="left" style="float: left">
 
       <h5 title="Mapping">
-        Mapping: <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+        Mapping: <span ng-show="rule.mapping">
+            <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+            <!--<a href="/releases/#{{ rule.mapping }}"><b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b></a>-->
+            <a href="" ng-show="rule.mapping" ng-click="openReleaseDataModal(rule.mapping)" class="glyphicon glyphicon-eye-open" title="View"></a>
+            <a href="/releases/#{{ rule.mapping }}" class="glyphicon glyphicon-pencil" title="Edit"></a>
```

Comment:
Does this work?! If so, I clearly don't remember how this app works :)
I know this app is using ngRoute which is the old way of doing pushState routing. There's newer and better module called [`ui-router`](https://github.com/angular-ui/ui-router) which started out as a plugin and has since been blessed as thee standard routing library for angular 1x. 

However, this link here I think does a whole new refresh of the single-page app and starts at `/releases/` this time. 
Isn't there a function you can use to go to a different route?